index.js

```diff
@@ -168,11 +174,11 @@ MockDate.now = _Date.now;
 MockDate.UTC = _Date.UTC;
 
 MockDate.prototype.toDateString = function () {
+  if (this.invalid) return new _Date('').toDateString();
   return `${ weekDays[this.getDay()] } ${ months[this.getMonth()] } ${ this.getDate().toString().padStart(2, '0') } ${ this.getFullYear() }`;
 };
 
 // TODO:
-// 'toDateString',
```

Comment:
I think this TODO item is done?

Reply:
Sure is!
`// 90% of all comments are wrong`